service/dap/server.go

```diff
@@ -1522,21 +1522,27 @@ func (s *Session) setBreakpoints(prefix string, totalBps int, metadataFunc func(
 				err = errors.New("breakpoint already exists")
 			} else {
 				bp := &api.Breakpoint{
-					Name:    want.name,
-					File:    wantLoc.file,
-					Line:    wantLoc.line,
-					Addr:    wantLoc.addr,
-					Addrs:   wantLoc.addrs,
-					Cond:    want.condition,
-					HitCond: want.hitCondition,
+					Name:         want.name,
+					File:         wantLoc.file,
+					Line:         wantLoc.line,
+					Addr:         wantLoc.addr,
+					Addrs:        wantLoc.addrs,
+					Cond:         want.condition,
+					HitCond:      want.hitCondition,
+					DidUnsuspend: s.didUnsuspendBreakpoint,
 				}
 				err = setLogMessage(bp, want.logMessage)
 				if err == nil {
 					// Create new breakpoints.
-					got, err = s.debugger.CreateBreakpoint(bp, "", nil, false)
+					got, err = s.debugger.CreateBreakpoint(bp, "", nil, true)
```

Comment:
I think this should work like the command line client does: only create suspended breakpoints if the plugin package is present or if follow exec is enabled (https://github.com/go-delve/delve/blob/be3019a9eb16255d0e170cdd7bf65e16e49e998c/pkg/terminal/command.go#L1862).

About the notifications I have commented on the issue.